packages/datagateway-download/src/downloadCart/downloadCartApi.ts

```diff
@@ -45,10 +49,17 @@ export const removeAllDownloadCartItems: () => Promise<void> = () => {
 
 export const removeDownloadCartItem: (
   entityId: number,
-  entityType: string
-) => Promise<void> = (entityId: number, entityType: string) => {
+  entityType: string,
+  facilityName: string,
+  downloadApiUrl: string
```

Comment:
If you agree with my suggestion that the settings are clearer if supplied via an object, this change should be applied to all functions to have a uniform interface

Reply:
@louise-davies I have addressed this in https://github.com/ral-facilities/datagateway/pull/114/commits/df8ef1f93d22a7ac57d54ddbc87bf23648ba0060.